exec-workitem/src/test/java/org/jbpm/process/workitem/exec/ExecWorkItemHandlerTest.java

```diff
@@ -85,6 +85,35 @@ public void testExecCommandWithArguments() throws Exception {
         assertNotNull(result);
         assertTrue(result.contains("java version") || result.contains("jdk version"));
     }
+    
+    @Test(timeout=6000)
```

Comment:
I dont understand how this test method actually tests the code that you have added in the handler :) please explain. Need a test that actually times out to test that the timeout is actually handled.

Reply:
In this test I am executing ping command, when we execute ping command from the terminal we have to manually terminate the command. Here junit test will wait for 5 seconds to complete its executiom. If command execution didnt terminated/completed within 5 secs, test will keep running for its completion. So to terminate junit test added 6 seconds timeout for test.